src/view/UserphraseView.cpp

```diff
@@ -75,6 +79,12 @@ void UserphraseView::addPhrase(int result)
     }
 
     emit model()->add(phrase, bopomofo);
+    
+    if (dialogType_ == DIALOG_MODIFY) {
+	if (model()->sourceModel()->getAddResult() <= 0) {
```

Comment:
I think you shall ensure the indent is the same. Since this project has `.editorconfig`, you can find a suitable plugin for your editor in <http://editorconfig.org/>.